charts/mockup-station-cadip/templates/configmap.yaml

```diff
@@ -25,8 +25,12 @@ metadata:
 data:
   {{ .Values.app.authConfigFile }}: |-
     {
-        "test": "$2b$12$MfKwr.11KN7OjCqLLwh/yuLhXasqSzTTdsz71YlnoeZdqlELqlyQi",
-        "ovidiu": "$2b$12$KNiiNLAIbJDj/ftcVidt1.mVxSn4rttIByudz9GAx5IqPEsWg38jW"
+        "client_id": "client_id",
+        "client_secret": "client_secret",
+        "username" : "test",
```

Comment:
maybe not that important, but you could use b64encrypt here, as before

Reply:
not that important, it's just a mockup for tests